delete_non_clonable.js

```diff
@@ -78,16 +78,18 @@ function deleteNonClonable(obj, depth) {
         }
         case 'XMLHttpRequest':
             var newXHR = {
-                readyState: obj.readyState,
-                responseText: obj.responseText,
-                status: obj.status,
-                statusText: obj.statusText
+                  readyState: obj.readyState,
+                  status: obj.status,
+                  statusText: obj.statusText
+            };
+            if (obj.reponseType !== 'blob') {
```

Comment:
Looks like the error could happen for other responseType too, for example `'arraybuffer'`. Should we limit to only known supported ones (`''` and `'text'`) instead?